contrib/platform/src/com/sun/jna/platform/win32/Crypt32.java

```diff
@@ -43,6 +43,63 @@
 
     Crypt32 INSTANCE = Native.load("Crypt32", Crypt32.class, W32APIOptions.DEFAULT_OPTIONS);
 
+    // Store providers for CertOpenStore
```

Comment:
This should go into WinCrypt.java as definition comes from WinCrypt.h. The two encoding definitions are already there.

Reply:
The new constants should still be moved to `WinCrypt.java`. As `CertStoreProviderName` is not from a header it is debatable, but I would also move it.

Reply:
Done. I did not add a javadoc to each of them, from MSDN page it looks like a lot of text with references and we probably don't want to have it all copied to WinCrypt